FordLabs/PeopleMover
# Recognize user in space header

## Issue
Connects #401 

## What was done
- [x] Added user cdsid to the dashboard account dropdown toggle
- [x] Refactored header and account dropdown

<img width="1111" alt="Screen Shot 2020-09-29 at 2 01 05 PM" src="https://user-images.githubusercontent.com/17144844/94597944-3f77e180-025c-11eb-9acd-598dc9645378.png">


## How to test
1. Login and go into a space. Ensure your cdsid shows up on the right side of the header next the the account dropdown.
2. Ensure the actions in the action dropdown work properly still.

NOTE: if you go to the flipping sweet board, there is no current user, so nothing will show.
